### DISCUSSION TITLE
Use the code autocomplete in VS Code

### DISCUSSION THREAD

**Discussion by tttsss-01** [idaholab/moose#24153] [Q&A General] [answered]
Hello,

I am trying to use the code autocomplete in VS Code. I have attempted to use the MOOSE Language Support extension, but it keeps displaying the error message "No MOOSE application executable found". Where should I provide the path for the MOOSE application? I have noticed in the extension settings that I can provide a HIT format style file, but I am not sure what HIT is and how to provide it. Will providing this file make the extension work? My VS Code is running on Windows and connected to WSL. I am not very familiar with VS Code. Can someone please help me?
![image](https://user-images.githubusercontent.com/110894574/233686836-9d0ef936-dbfe-4907-b1a6-2c7312419308.png)

**Comment by GiudGiud** [comment, marked as answer]
Hello

if you are working in the repository of the application, like in the tests or examples folder for example, it should find it automatically 

if not then you should create a symlink to the executable in your local working folder

Guillaume

**Reply by tttsss-01** [reply]
This worked. Thank you very much.
